_topic_map.yml

```diff
@@ -575,6 +575,9 @@ Topics:
 - Name: Pruning Objects
   File: pruning_resources
   Distros: openshift-*
+- Name: Extending the Kubernetes API with Custom Resources
+  File: custom_resource_definitions
+  Distros: openshift-*
```

Comment:
@mburke5678 Do we know if this is relevant for OSD?

Reply:
@mburke5678 @adellape - I am not sure that this is relevant to OSD or Online. Michael has marked the label with enterprise only but the topic map makes it for all. I am going to remove it from dedicated and online till we know for sure.